dev-docs/windows-dev-setup.md

```diff
@@ -0,0 +1,462 @@
+src\mewbot\ioconfigs\post_input\input.py:21: error: Cannot find implementation or library stub for module named "mewbot.io"
+Found 4 errors in 4 files (checked 15 source files)
+```
+
+(These errors are to be expected - mewbot is not currently installed in the venv - so the static analysis tools - which are using the venv's python.exe get confused.)
+NOTE - The number of files checked should match the number of files which where checked by black.
+
+```shell
+(mewbot_venv) C:\mewbot_dev\mewbot>pylint src
+```
+
+(This will probably throw a bunch of errors - we're just testing the venv for now).
+
+9) Install mewbot in development mode
+
+```shell
+(mewbot_venv) C:\mewbot_dev\mewbot>python setup.py develop
+```
+
+(`develop` instead of `install` means that the interpreter will use the files here instead of copying them elsewhere.
+If you accidentally do `install` not `develop` ... may be best to delete the venv and start again).
+
+10) Test an example actually works
+
+It's a case of "bring your own token" - instructions as to how to generate tokens for each of the supported services are located elsewhere in this documentation.
+
+Execute something like
+
+```shell
+(mewbot_venv) C:\mewbot_dev\mewbot>python -m examples examples\discord_to_desktop_notification.yaml
+```
+
+with the paths replaced as apporpiate for your setup and the example replaced with the example you actually want to run (though the desktop notificaiton example is perfectly inoffensive).
+
+11) Re-run linters
+
+With mewbot actually installed, the linters should now actually all work.
+
+Confirm it by running them against the `src` folder
+
+```shell
+(mewbot_venv) C:\mewbot_dev\mewbot>black src
+All done! ✨ 🍰 ✨
+15 files left unchanged.
+```
+
+e.t.c.
+
+12) Check you can run linters through the linting script
+
+The previous tests have mostly to make sure that the linters are all installed, before running them as intended during dev.
+
+To do this the venv path must be modified to include the root of the checked out source.
+In my case
+
+```shell
+C:\mewbot_dev\mewbot
+```
+
+There are a number of ways to do this.
+
+#### Use a .pth files
+
+Probably the best way is to use a `.pth` file - these provide instructions to the python interpreter as to additional folders to include in its path.
+
+A path file is just a text file with extension `.pth` which include folders you want added to the python path - one per line.
+
+It must be placed on the path of the python interpreter your currently using. In this case, assuming you have the same paths as me, probably the best place to put it would be
+
+```shell
+C:\mewbot_dev\mewbot_venv\Lib\site-packages
+```
+
+Simple add a file named `mewbot.pth` here with the single line `C:\mewbot_dev\mewbot`. It's probably best to add it as a new file, instead of modifying one of the existing ones - as these are under system control and may be overwritten without warning.
```

Comment:
```suggestion
Simple add a file named `mewbot.pth` here with the single line `C:\mewbot_dev\src`. It's probably best to add it as a new file, instead of modifying one of the existing ones - as these are under system control and may be overwritten without warning.
```
Pretty sure this should be as suggestion.

Reply:
Corrected